src/reference/asciidoc/whats-new.adoc

```diff
@@ -14,3 +14,13 @@ development process.
 === General Changes
 
 ==== Core Changes
+
+==== JMS Changes
+
+Previously, Spring Integration used a default bean name `connectionFactory` for the JMS Connection Factory, allowing the property to be omitted from component definitions.
+
+Spring Boot auto-configures a bean name `jmsConnectionFactory`.
+
+Since it is common to use Spring Integration XML configuration with Spring Boot, it was decided that it would be useful to align the two so that an SI app can use the auto configured bean.
```

Comment:
Polishing for your review: https://github.com/artembilan/spring-integration/commit/4dfc8517e4fc9e89b662bb82a8be67dfd28a6c29

Somehow I don't like your language in this sentence, plus this your promotion for XML config with Boot doesn't help our Java & Annotation Configuration story :smile: